src/acparser/__init__.py

```diff
@@ -44,54 +47,85 @@ def system(cmd):
+    parser.add_argument(
+        "--namespace", help="Namespace of the collection", required=True
+    )
+    parser.add_argument("--name", help="Name of the collection", required=True)
+    parser.add_argument("--version", help="Version of the collection", required=True)
+    args = parser.parse_args()
+    namespace = args.namespace
+    collection_name = args.name
+    collection_version = args.version
+
+
+    EXISTS_GALAXY = False
+    REQUIRES_ANSIBLE = ""
+    LICENSE = ""
+    CHANGELOG = ""
+    REQUIREMENTS = []
+    COMMUNITY_COLLECTIONS = ""
+
+
     with tempfile.TemporaryDirectory() as collection_dir:
-        print("Downloading collection... \n")
-        system(
+        _, _, retcode = system(
             f"ansible-galaxy collection download -n -p {collection_dir} {namespace}.{collection_name}:{collection_version}"
         )
-        print("Collection downloaded. \n")
-        tarfilename = os.path.join(
-            collection_dir, f"{namespace}-{collection_name}-{collection_version}.tar.gz"
-        )
+        # check the return code
+        if retcode == 0:
+            tarfilename = os.path.join(
+                collection_dir,
+                f"{namespace}-{collection_name}-{collection_version}.tar.gz",
+            )
+            if os.path.exists(tarfilename):
+                EXISTS_GALAXY = True
+
+    # printing the output
+    if EXISTS_GALAXY:
+        print("Source exists in galaxy.")
+    else:
+        print("Source does not exist in galaxy.")
+
+    tarfilename = args.tarfile
+
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        # Extract the tarball
+        export_tar(tarfilename, tmpdirname)
 
-        with tempfile.TemporaryDirectory() as tmpdirname:
-            # Extract the tarball
-            export_tar(tarfilename, tmpdirname)
+        # check runtime ansible version
+        runtime_yml = f"/{tmpdirname}/meta/runtime.yml"
+        if os.path.exists(runtime_yml):
+            with open(runtime_yml, "r") as fobj:
+                data = yaml.load(fobj, Loader=yaml.SafeLoader)
+                REQUIRES_ANSIBLE = data['requires_ansible']
+                print(
+                    f"{namespace}.{collection_name}:{collection_version} requires ansible-core version {data['requires_ansible']}"
+                )
+        else:
+            print("runtime.yml does not exists")
 
-            # check runtime ansible version
-            runtime_yml = f"/{tmpdirname}/meta/runtime.yml"
-            if os.path.exists(runtime_yml):
-                with open(runtime_yml, "r") as fobj:
-                    data = yaml.load(fobj, Loader=yaml.SafeLoader)
-                    print(
-                        f"{namespace}.{collection_name}:{collection_version} requires ansible-core version {data['requires_ansible']}"
-                    )
-            else:
-                print("runtime.yml does not exists")
+        print("")
```

Comment:
```suggestion
        print()
```

This can be simplified to `print()`